Fix get nil task crash

MengLeiFudge opened the pull request:
## Fix get nil task crash
### Goal
Fixed a bug in multiplayer mode involving the use of the to-do list that caused the game to crash immediately.

### Questions/Feedback request
<img width="1648" height="1042" alt="DF4200688B00056AED2A0626A9CAE605" src="https://github.com/user-attachments/assets/6fa878ec-6817-400c-b2a6-8d06c024770e" />
Following testing, we hypothesise this bug arises as follows:
1. The to-do list already contains task1, then both Player A and Player B open the window
2. Player A deletes task1 and clicks save
3. Player B clicks save
At this point, the game encounters a nil error because task1 cannot be found for Player B, subsequently causing the game to crash
I am not particularly familiar with the todo-list code; if you have a better method for fixing this, that would be most welcome.

Alexander-Serov commented:
Hey @MengLeiFudge! Thanks for reporting and providing a first solution. I will have a look. Let's also see what the copilot thinks.

Alexander-Serov commented:
@copilot Check the issue appearing in the multiplayer game when attempting to delete/interact with the same issue. Check the proposed solution. As suggested it makes sense to skip the issue if not present, but maybe show a message to the user that another player has already removed the issue? 
Also, update the version in the info.json and the changelog so that the pipelines pass.

Alexander-Serov commented:
@MengLeiFudge Looks good to me. Have a look at a couple of comments above, does the suggestion make sense to you?

Also, for the pipelines to pass you should update the version in info.json and the changelog.

Once the pipelines pass, they will create a zip with a new version of the mod. Would you be so kind as to download it with your friend and test if you can recreate the error?

Thanks so much for reporting!

MengLeiFudge commented:
Right. Regarding the changes mentioned by the AI, I think adding some player-visible prompts during conflicts is a sound approach. I shall first test whether this issue reoccurs in the new version. Once confirmed, I'll push out a revised build number and add the change log—though that might have to wait until tomorrow, as my mates have all turned in for the night:) Finally, thank you for your prompt reply!

Alexander-Serov commented:
No worries and no hurry! Looking forward to seeing it fixed!

MengLeiFudge commented:
Hello! Following testing, the game no longer crashes. I've adjusted the version number and added the update log. I noticed some omissions in the Chinese translation, so I've supplemented those. Additionally, I observed that ‘Sort’ hadn't been translated. I've now implemented the language file's translation for this term and added it to all current language files to prevent key-finding errors. I have split them into several separate commits, and I would appreciate your review.

Alexander-Serov commented:
No, don't worry. Still getting used to it 😀 I will have a look myself anyway soon.
